inc/class-custom-walker-nav-menu.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Navigation menu walker class
+ *
+ * @package uds-wordpress-theme
+ */
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+/*
+ * Class Name: Custom_Walker_Nav_Menu
+ * Description: A custom WordPress nav walker class to customize navigation menu element wrapper and child menu's wrapper.
```

Comment:
Update description: A custom Nav Walker class to generate ASU Web Standards-compliant menus for widget-based menus (sidebars, etc.)